dataland-frontend/tests/e2e/support/index.ts

```diff
@@ -1,7 +1,8 @@
 import "@cypress/code-coverage/support";
```

Comment:
Do you know what mechanism guarantees that the `befireEach()` block in this file here is executed by cypress before the test-specs are executed?

Reply:
Ok cool thanks for the link+explanation.

And the `Commands.ts` file (which creates our custom commands ) runs because it is being imported by our support file, correct?

Because `Commands.ts` is not referenced anywhere else in the code.